### ISSUE TITLE
[FEATURE REQUEST] usability SD card free lookup/display in scripting?

### ISSUE THREAD

**Issue by clach04** [d0k3/GodMode9#691]
**Describe the feature you'd like**

Support to display:

* size of SysNAND
* current free disk space on SD card (via scripts)

Megascript for backing up sysnand https://github.com/d0k3/GodMode9/blob/master/resources/gm9/scripts/GM9Megascript.gm9#L28 recommends having enough free storage space on the SD card. But no concrete specifics.

**Describe alternatives you've considered**

* Update the backup script to display short help text to go back to the main screen to see SD free space.
* Unclear on how to determine SysNAND size.

**Comment by clach04** [comment]
I posted a quick PR that attempts the later option rather than the feature request.

This would have helped me greatly if it had been present.

Thanks for making gm9 available!

**Comment by d0k3** [comment]
Thanks! I'll look into it later and will merge or get back about any issues!

**Comment by d0k3** [comment]
Okay, I've got something for you. Scripting now has $[SDSIZE], $[SDFREE] and $[NANDSIZE] global variables. I guess you could improve on your commit using these. Take not, however, that on large SD cards, calculating the free space can take some time. You need to test and account for this.

Here's the build with the new possibilities:
https://f.secretalgorithm.com/12cbfv/godmode9.firm

**Comment by clach04** [comment]
@d0k3  Works great! 👍 Thanks for doing this.

Managed to grab a few mins to play with it. On my 64Gb card, I did not notice any speed penalties. :)

I posted a comment on https://github.com/d0k3/GodMode9/commit/0825139cb2fcda3ec6de13b3b7590eabd517ad54#commitcomment-49067727

I have a dumb test at https://github.com/clach04/GodMode9/commit/d5ae9839bb0d878dbdea26c8a1833f645765c4bb which works for me (but the previous dialog question is still the same, will try and find time to clean this up)
